test/functional/Valhalla/playlist.xml

```diff
@@ -63,6 +66,8 @@
 			  -->
 			<variation>-Xjit:count=1,disableAsyncCompilation -Xgcpolicy:optthruput</variation>
 			<variation>-Xjit:count=1,disableAsyncCompilation -Xgcpolicy:optthruput -Xnocompactgc -XX:ValueTypeFlatteningThreshold=99999</variation>
+			<variation>-Xjit:count=1,disableAsyncCompilation -Xgcpolicy:gencon -Xnocompactgc </variation>
```

Comment:
same here

Reply:
As in we want the variation with `-XX:ValueTypeFlatteningThreshold=99999`, or remove it entirely?

Reply:
no I mean add `-Xjit:count=1,disableAsyncCompilation -Xgcpolicy:gencon -Xnocompactgc  -XX:ValueTypeFlatteningThreshold=99999` but not `-Xjit:count=1,disableAsyncCompilation -Xgcpolicy:gencon -Xnocompactgc`.

The GC is only affected when arrays are flattened. We use the other modes to test JIT and VM behviour.

Reply:
nitpick, there is a extra space here `-Xnocompactgc  -XX:ValueT`